### ISSUE TITLE
Installer is not working

### ISSUE THREAD

**Issue by JoeSmurf98** [tgraupmann/MinecraftChromaMod#1]
I have both the latest recommended forge mod loader installed and I have run your installer. 

When I check my minecraft mods folder your files have not been transferred across. 
Neither windows defender or AVG seem happy with the .exe so these may be causing an issue. Any help would be appreciated or a link to the files so I can just drop them into the appropriate folder would be great

**Comment by SkyroOryks** [comment]
mine installed, but forge does not see the mod

**Comment by Torbikini** [comment]
Yeah, I ran it but there's no mod in the mods folder. Can you just release the jar file? I don't see why there's an installer.

**Comment by Depresst0** [comment]
yeah i got mc open but chroma isnt seeing it so idek what to do anymore
might just download Synapse 3 and see what happens, cause it seems to do something with synapse as well as chroma, so it might be bc im running synapse 4 and standalone chroma.
